.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject/
```

Comment:
Why is this line added? I think that this line belong to your global settings `.gitignore`.

Thanks